internal/pkg/v2/infrastructure/redis/reading.go

```diff
@@ -26,6 +26,9 @@ const (
 	ReadingsCollectionName       = ReadingsCollection + ":" + v2.Name
 )
 
+// deleteReadingsByIdChannel channel used to delete readings asynchronously
+var deleteReadingsByIdChannel = make(chan []string, 50)
```

Comment:
Move to Core Data app layer

Reply:
Per comments from Cloud above, I still leave this channel in the infra layer.